src/app/logic/player.js

```diff
@@ -1,23 +1,37 @@
+// eslint-disable-next-line import/extensions
```

Comment:
Consider adding this as a rule for eslint to ignore in all files

Reply:
good idea! will be added to rules, but maybe outside this PR